src/main/java/net/glowstone/net/message/play/game/TitleMessage.java

```diff
@@ -41,9 +41,12 @@ public TitleMessage(Action action) {
     }
 
     public static TitleMessage[] fromTitle(Title title) {
+        TextMessage titleMessage = title.getTitle() != null ? asTextMessage(BaseComponent.toPlainText(title.getTitle())) : asTextMessage(null);
+        TextMessage subTitleMessage = title.getSubtitle() != null ? asTextMessage(BaseComponent.toPlainText(title.getSubtitle())) : asTextMessage(null);
```

Comment:
After looking into `Optional`s, I think that md5's api would need that. It is `BaseComponent.toPlainText` that is requiring us to do this. When `title.getTitle` or `title.getSubTitle` is null, it creates a `NullPointerException` in the md5 api.